.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+
+jobs:
+  test:
+    name: 테스트 코드 실행
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 체크아웃
+        uses: actions/checkout@v3
+
+      - name: JDK 버전 설정
+        uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)